pom.xml

```diff
@@ -95,6 +95,26 @@
         <pax.cdi.version>0.12.0</pax.cdi.version>
         <validation.version>1.1.0.Final</validation.version>
         <commons-configuration.version>1.10</commons-configuration.version>
+        <servicemix.bundles.mina.version>1.1.7_6</servicemix.bundles.mina.version>
+        <qpid-jms-client.version>0.6.0</qpid-jms-client.version>
+        <proton-j.version>0.10</proton-j.version>
+        <netty-all.version>4.0.17.Final</netty-all.version>
+        <geronimo-jta_1.1_spec.version>1.1.1</geronimo-jta_1.1_spec.version>
+        <geronimo-jms_1.1_spec.version>1.1.1</geronimo-jms_1.1_spec.version>
+        <servicemix.bundles.freemarker.version>2.3.23_1</servicemix.bundles.freemarker.version>
+        <servicemix.bundles.commons-httpclient.version>3.1_7</servicemix.bundles.commons-httpclient.version>
+        <commons-pool.version>1.6</commons-pool.version>
+        <servicemix.bundles.junit.version>4.11_2</servicemix.bundles.junit.version>
+        <!-- Artemis dependencies -->
+        <artemis.version>1.3.0</artemis.version>
+        <proton.version>0.12.2</proton.version>
+        <geronimo-jms_2_spec.version>1.0-alpha-2</geronimo-jms_2_spec.version>
+        <jboss-logging.version>3.3.0.Final</jboss-logging.version>
+        <jgroups.version>3.6.9.Final</jgroups.version>
+        <commons-beanutils.version>1.9.2</commons-beanutils.version>
+        <netty.version>4.0.32.Final</netty.version>
+        <qpid-jms.version>0.9.0</qpid-jms.version>
+        <commons.fileupload.version>1.3.2</commons.fileupload.version>
```

Comment:
Yes I cherry-pick'd this one. Had issues which I had to manually resolve and the server claimed these should be part of it. 

I'm going to remove them and try to build.